src/main/java/es/unizar/smartcampuz/application/controller/UserController.java

```diff
@@ -1,27 +1,26 @@
 package es.unizar.smartcampuz.application.controller;
```

Comment:
This class must be renamed. Moreover, you  have to delete the remaining methods discussed in the chat group.